setup.pl

```diff
@@ -674,11 +697,26 @@
 		print RELOADD "$systemctlcmd reload webmin\n";
 		close(RELOADD);
 
+		# Pre install
+		open(PREINSTT, ">$config_directory/.pre-install");
```

Comment:
N: This section generates pre and post install scripts for newer systems with `systemd` to use _SIGSTOP_ and _SIGCONT+SIGHUP_ respectively and only on the _main_ process. We can not leave it default to _all_ as it would freeze the whole control group, making updates from UI itself frozen too.